### PR TITLE
[hi] Remove Katacoda mentions in tutorials

### PR DESCRIPTION
**Pull request by utkarsh-singh1**
Description:
Removed katacoda instances in kubernetes-tutorials

Page to Update:
https://kubernetes.io/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive/

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
fixes - #41496

**Review by aj11anuj — COMMENTED** [review]
LGTM, 👍

File: content/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html [hunk 1]
```diff
@@ -15,20 +15,20 @@
...
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
+
+                अधिक जानकारी के लिए देखें <a href="https://kubernetes.io/pt-br/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
```
Comment (Gauravpadam) [inline content/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html:27]: This should redirect to "https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/"

**Review by Gauravpadam — COMMENTED** [review]
Thanks,

One thing, I would redirect to the English blog article

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html [hunk 2]
```diff
@@ -15,33 +15,20 @@
...
-                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">मॉड्यूल 3 पर जाए &gt;<span class=""></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```
Comment (Gauravpadam) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:25]: Better to localize these in correct contexts OR we could do this
```suggestion
इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```

File: content/hi/docs/tutorials/kubernetes-basics/explore/explore-interactive.html [hunk 3]
```diff
@@ -15,25 +15,20 @@
...
-                <a class="btn btn-lg btn-success" href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/" role="button">मॉड्यूल 4 पर जाएं &gt;<span class="btn"></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```
Comment (Gauravpadam) [inline content/hi/docs/tutorials/kubernetes-basics/explore/explore-interactive.html:25]:
```suggestion
इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```

File: content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 4]
```diff
@@ -15,22 +15,20 @@
...
-                <a class="btn btn-lg btn-success" href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/" role="button">मॉड्यूल 5 पर जाएं &gt;<span class=""></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```
Comment (Gauravpadam) [inline content/hi/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:25]:
```suggestion
इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```

File: content/hi/docs/tutorials/kubernetes-basics/scale/scale-interactive.html [hunk 5]
```diff
@@ -15,27 +15,23 @@
...
-                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/update/update-intro/" role="button">मॉड्यूल 6 पर आगे बढ़ें &gt;<span class=""></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```
Comment (Gauravpadam) [inline content/hi/docs/tutorials/kubernetes-basics/scale/scale-interactive.html:25]:
```suggestion
इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```

File: content/hi/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 6]
```diff
@@ -15,21 +15,21 @@
...
-               <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">मुलभुत कुबेरनेट्स पर लौटें<span class=""></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```
Comment (Gauravpadam) [inline content/hi/docs/tutorials/kubernetes-basics/update/update-interactive.html:25]:
```suggestion
इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```

**Review by Gauravpadam — COMMENTED** [review]
Thanks for the wider set of changes,

A few things,

- Try to localize with context
- If not, The suggested changes are fine too!

File: content/hi/docs/tutorials/hello-minikube.md [hunk 7]
```diff
@@ -15,14 +15,7 @@ card:
...
 
-यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए 
-कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है।
-काटाकोडा आपके ब्राउज़र पर मुफ़्त कुबेरनेट्स वातावरण प्रदान करता है।
-
-{{< note >}}
-यदि आपने अपने स्थानीय सिस्टम पर मिनीक्यूब स्थापित किया है तो आप इस ट्यूटोरियल का अनुसरण कर सकते हैं।
-स्थापाना निर्देश के लिए [मिनीक्यूब पृष्ट](https://minikube.sigs.k8s.io/docs/start/) देखें। 
-{{< /note >}}
+यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है। ट्यूटोरियल एक कंटेनर छवि प्रदान करता है जो सभी अनुरोधों को प्रतिध्वनित करने के लिए NGINX का उपयोग करता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/tutorials/hello-minikube.md:18]: Doesn't reflect the current version of the Hello Minikube document

https://kubernetes.io/docs/tutorials/hello-minikube/.

Kindly make appropriate changes.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by sftim** [comment]
/retitle [hi] Remove Katacoda mentions in tutorials

**Comment by dipesh-rawat** [comment]
Hey @utkarsh-singh1, just dropping a friendly note to see if you have the capacity to tackle this. This pull request requires a rebase against the main branch to resolve conflicts.

**Comment by utkarsh-singh1** [comment]
Hi @dipesh-rawat, I have rebased the branch and updated the PR.

**Comment by utkarsh-singh1** [comment]
Hi @divya-mohan0209, I have make changes to the PR can can you review it.

**Comment by utkarsh-singh1** [comment]
Hi @dipesh-rawat @divya-mohan0209, Are there any more changes required in the PR?

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for these changes!

/approve

**Comment by utkarsh-singh1** [comment]
Thanks @dipesh-rawat @divya-mohan0209 for the review.

**Pull request merged**